### PR TITLE
Use new tags in pipeline task defs

### PR DESCRIPTION
**Pull request by kenoir**
Required for ECS deployment using the `weco-deploy` tool.

Depends on https://github.com/wellcomecollection/catalogue/pull/681

### CODE REVIEW

**Comment by kenoir** [comment]
For https://github.com/wellcomecollection/platform/issues/4676

File: .wellcome_project [hunk 1]
```diff
@@ -72,6 +72,63 @@ catalogue_pipeline:
       name: Production
     - id: miro_merging_test
       name: Miro merging test
+  image_repositories:
+    - id: id_minter
+      namespace: uk.ac.wellcome
```
Comment (jamieparkinson) [inline .wellcome_project:77]: Is it worth using an alias/anchor for the repeated account ID and namespace keys?

Comment (kenoir) [reply .wellcome_project:77]: It's worth doing something to reduce repetition (as previously mentioned by @alexwlchan). I'm not a massive fan of references in YAML. I think the deploy tool should provide a mechanism to specify (and override) defaults.

Comment (jamieparkinson) [reply .wellcome_project]: yeah that sounds like a better idea - also not a huge fan of YAML references

**Pull request merged**